src/api/goods.ts

```diff
@@ -1,11 +1,29 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
+function wait(delay: number) {
+  return new Promise(resolve => setTimeout(resolve, delay));
+}
+
+export async function getAll(): Promise<Good[]> {
+  await wait(2000);
+
   return fetch(API_URL)
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = async () => {
+  const [allGoods] = await Promise.all([
+    getAll(),
+  ]);
```

Comment:
Hmm, I don't think it `Promise.all()` does anything useful in this case😅 Promises should be combined in one `Promise.all()` for it to work (Send multiple requests at the same time)

Reply:
i agree, just wanted to try it but then realized it doesn't do its job here